### DISCUSSION TITLE
Plans with no underlying contracts

### DISCUSSION THREAD

**Discussion by jwerner15083** [CMSgov/price-transparency-guide#497] [Q&A]
Fairly simple question, but have not received guidance on this yet.  If there is a plan that relies solely on RBP, meaning that they have no underlying provider contracts, are we required to put out a MRF with that data?  If so, do we provider any data in the provider_groups array?  I assume no, but since there is no specific tin or NPI that exists, there is no contract to pull from.  The rates are universal.  

Of course this creates an impossible situation when presenting rates based on a wide provider base.  

Sorry for asking this so late, I was hoping that it would have been sorted already.

**Comment by Labouv0506** [comment]
We have a few ?s on this as well. If there is no negotiated rate, then the rates would come from the allowed amounts file. In the instance we have multiple sources for the rates and %s applied to the rate as it will vary by plan. the oon mrf will not have all the codes and information needed. Pricing is done by a third party, but since these are not negotiated, there is no onus on them to supply rates. How have folks solved for this? as there is no contract or negotiated rates, and unless they have lots of examples the oon MRF will not supply either.

**Comment by shaselton-usds** [comment]
@jwerner15083 @Labouv0506 Just wanted to check if this was the scenario you're referring to: https://www.federalregister.gov/d/2020-24591/p-646

**Reply by jwerner15083** [reply]
@shaselton-usds Yes it answers it for the RBP side.

There still is an outstanding question on why the regs do not allow a more simplified MRF for reference based contracts similar to how the percentage of charges was allowed.  In the % of charge situation the charge master was deemed too difficult to "understand or create" so a detailed line by line output was dropped in favor of a much simpler X % of charges output.

Contracts based on a % of CMS values such as IPPS, Professional PPS, DME, etc...all provided under the CMS "umbrella" could also be vastly simplified by allowing this same model.  Instead of (for example) listing every DRG with the expected rate, we could simply state that it is 150% of the Acute IPPS system.   That would also allow the documentation of the RBP plans as well (no contract plans) because the fluctuating rates that cannot be "displayed" due to the underlying provider/location based variable would then be allowed.  You could simply state "we pay all Acute IP claims at 185% of CMS"...done.

Doing that also resolves things such as APC bundling and multiple procedure rate structures that are going to cause havoc with the rate structure.  Today I have to present a value for every code, even those that would never be billed alone or would "always" be bundled under CMS guidelines.  A member looking at that would increase their expectations ,causing invalid responses to the 1/1/2023 price estimation tool.  

But yes this confirms that RBP only plans (no contracts) would not need to create a MRF for those plans.

**Comment by Labouv0506** [comment]
@shaselton-usds    yes it answers for the mrf..(which is part of the problem)..but what about the requirement to show Member liability in the cost tool. with no rate on an MRF and no onus to supply (in the case of a third party vendor) what are we to show? we can disclaim it...but we will never be able to show Member liability accurately as two items will be missing: the RBP rate, and the billed amount?
any thoughts?
